python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/port_channel_interfaces.schema.yml

```diff
@@ -392,6 +392,13 @@ keys:
             - str
           min: 0
           max: 300
+        min_links:
```

Comment:
It would be better to use key `links_minimum` . For minimum links the key `links_minimum` is used in `link_tracking_groups` data model.

Reply:
Nevermind, key name is identical to command.